### ISSUE TITLE
PaperMC server the dynmap broke with latest PaperMC update

### ISSUE THREAD

**Issue by RaidersPlay** [webbukkit/dynmap#2938]
Upgrading from PaperMC 264 to 299 the dynmap rendering somehow broke.
Seems like only where players was the maps got updated and the rendering created black blocks.
I've downgraded the server back to 264 and executed a radiusrender again.

I've not pickup any other errors on the other Spigot plugins I'm running.
I'm running Dynmap v3.0-beta-10, checked if there was an updated Dynmap but didn't seen any.

![image](https://user-images.githubusercontent.com/58973538/82235228-bad29980-9932-11ea-9693-ba6ffb01efb3.png)

**Comment by RaidersPlay** [comment]
I've tried downgrading, it seemed to have fixed a few tiles, but others started to corrupt. Thus giving me the idea that something else is up.
I've searched for an error in logs, and found nothing.

**Comment by pizzanakin** [comment]
I have a similar issue, but for me it happened after switching the server to a different system. Although the new system has automatic paper updates set up, which means it could in fact have been caused by one of those.

Check out this issue:
https://github.com/webbukkit/dynmap/issues/2822

**Comment by naughtius-maximus-8492** [comment]
Tuinity user here, I had a perfect map render of my world then build 99 of Tuinity came out which also included the latest upstream build of paper. Started getting these tiles turning black and every time I viewed the world, it got worse. /fullrender's didn't work, tried wiping the tiles and starting off from scratch didn't work.

Eventually, I reverted back to build 96 of Tuinity which is what I was using before and deleted the tiles and rendered again. Everything is perfectly fine now. Seems to be an issue with one of the latest builds of paper although I'm not sure which is the latest safe build to use.
